pytest_fixtures/core/contenthosts.py

```diff
@@ -21,7 +21,9 @@ def host_conf(request):
     _rhelver = f"{distro}{params.get('rhel_version', settings.content_host.default_rhel_version)}"
     # check to see if no-containers is passed as an argument to pytest
     deploy_kwargs = {}
-    if not request.config.getoption('no-containers', None) and not params.get('no_containers'):
+    if not request.config.getoption('no_containers') and not request.node.get_closest_marker(
+        'no_containers'
+    ):
```

Comment:
@JacobCallahan Why use logical `and` here? Is the expectation for the user to specify the marker and the pytest option if they don't want to use containers? Maybe that selection process is also done somewhere else? Because I feel like we would want `no_containers` if either option or marker were present.

Reply:
Basically we are checking that the switch for spinning up host as containers **from all the possible places** is **ON** :)

I tested it and the condition is very valid !

Reply:
Thanks for clarifying! I wasn't exactly sure what the check was doing.